src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+  slug: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, slug }) => {
+  const parentsHandler = (parentName: string | null) => {
+    const parent = people.find(person => person.name === parentName);
+
+    if (parentName && parent) {
+      return (
+        <PersonLink person={parent} />
```

Comment:
_just keep in mind on next time_: it's better to not put any JSX or components inside functions